lib/rex/post/meterpreter/packet.rb

```diff
@@ -314,6 +316,15 @@ def initialize(type, value = nil, compress=false)
     end
   end
 
+  def _tlv_type_string(type)
+    # Try to regenerate the cache if cache is empty, or the type we are looking for is not present in the cache.
+    regenerate_tlv_types_cache if @@cached_tlv_types.nil? || @@cached_tlv_types.key(type).nil?
+
+    return @@cached_tlv_types.key(type).to_s.gsub('TLV_TYPE_', '').gsub('_', '-') unless @@cached_tlv_types.key(type).nil?
```

Comment:
Might need a join/map here if a TLV id could actually map to multiple values - and it's not worth the developer cycles right now investing time to make it smarter about detecting which command we're running, and which TLV the developer probably wants to see